templates/observability/index.html

```diff
@@ -0,0 +1,647 @@
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/cb930e58-2018-logo-cogeco.svg",
+            alt="",
+            width="145",
+            height="145",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/a2e4ca92-logo-rabobank.svg",
+            alt="",
+            width="144",
+            height="144",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/52df407b-2018-logo-fibernet.svg",
+            alt="",
+            width="145",
+            height="145",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/99d78ec1-2018-logo-biznet-giocloud.svg",
+            alt="",
+            width="145",
+            height="145",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/3e7e40cd-2018-logo-mimedia.svg",
+            alt="",
+            width="145",
+            height="145",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/8fe23d60-2018-logo-voidbridge.svg",
+            alt="",
+            width="145",
+            height="145",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/f853c09b-2018-logo-uct.svg",
+            alt="",
+            width="145",
+            height="145",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--bottom">
+  <div class="u-fixed-width">
+    <h2>Managed monitoring tools, anywhere</h2>
+    <p class="p-heading--four">You focus on your apps, we run the tools for you</p>
+    <p>Great monitoring tools enable you to greatly improve your applications, but are hard and time-consuming to run reliably and at scale. Let us run the best open source monitoring tools for you, so you can leverage them to continuously improve your own applications.</p>
+    <p><a href="/managed/contact-us" class="p-button--positive">Request a free deployment assessment</a></p>
+    <hr class="p-separator" />
```

Comment:
This separator feels a bit too large, as we are qualifying with the grid below why it is a good idea for the prospect to get their monitoring tools to be run by Canonical